### PR TITLE
fix DLQ integration tests

### PR DESCRIPTION
**Pull request by jsvd**
This PR changes the DLQ integration tests to not rely on default names of indices for writing data to ES.
Also ensures ES is started in a single node mode.

These changes remove some of the magic of the integration test, where we relied on a index having a template with a geoip field with geo data types already. In these commits we explicitly push a template and then use a message that clearly clashes with that template.

### CODE REVIEW

File: qa/integration/specs/dlq_spec.rb [hunk 1]
```diff
@@ -100,44 +102,41 @@ def generate_message(number)
...
     context 'with multiple pipelines' do
-      let(:message) { generate_message(100)}
       let(:pipelines) {[
           {
               "pipeline.id" => "test",
               "pipeline.workers" => 1,
               "dead_letter_queue.enable" => true,
-              "pipeline.batch.size" => 1,
-              "config.string" => "input { generator { message => '#{message}' codec => \"json\" count => 1000 } } filter { mutate { add_field => { \"geoip\" => \"somewhere\" } } } output { elasticsearch {} }"
+              "pipeline.batch.size" => 100,
```
Comment (andsel) [inline qa/integration/specs/dlq_spec.rb:110]: The downsize of the batch size is it functional to the test or to the fact that now it's single node cluster?

**Review by andsel — APPROVED** [review]
LGTM, only tiny doubt on the change of `pipeline.batch.size`

File: qa/integration/specs/dlq_spec.rb (hunk 1 above)
Comment (andsel) [reply qa/integration/specs/dlq_spec.rb:110]: don't mind my comment, I thought it passed from 100 to 1, but it's opposite and is reasonable to have a batch greater than the unit

Comment (jsvd) [reply qa/integration/specs/dlq_spec.rb:110]: yeah having size 1 was making these tests unnecessarily slow

**Pull request merged**